### PR TITLE
Update intro and flux example

### PR DESCRIPTION
**Pull request by py-rh**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -13,6 +11,38 @@
    <a class="github-button" href="https://github.com/run-house/runhouse/fork" data-icon="octicon-repo-forked" data-size="large" aria-label="Fork run-house/runhouse on GitHub">Fork</a>
    </p>
 
+Why Runhouse?
```
Comment (mkandler) [inline docs/index.rst:14]: I think we can drop this or move it below the intro sentence. Page layout look kind of odd when you go straight from H1 to H2 without any intro paragraph. Technically the sentence following is more "what" than "why anyway.

File: examples/flux/flux.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import runhouse as rh
```
Comment (mkandler) [inline examples/flux/flux.py:1]: Is this meant to be in /examples or is it just a file to accompany a page in /guides? If the former, I think it needs an intro paragraph and h1 in comments about this line. Potentially a bit odd to start with an `import` statement.

File: examples/stable-diffusion-xl-ec2/README.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ make sure our AWS credentials are set up:
 $ aws configure
 $ sky check
 ```
-We'll be downloading the Llama2 model from Hugging Face, so we need to set up our Hugging Face token:
```
Comment (mkandler) [inline examples/stable-diffusion-xl-ec2/README.md:28]: 👍

**Review by mkandler — APPROVED** [review]
Looks good! Comments aren't necessarily blocking but may be worth updating before anything is published on the website.

File: examples/flux/flux.py (hunk 2 above)
Comment (py-rh) [reply examples/flux/flux.py:1]: Guides + Github, not examples.

File: docs/index.rst (hunk 1 above)
Comment (py-rh) [reply docs/index.rst:14]: image
I am not sure but I thought it felt much weirder to launch into normal text without some kind of subheader?

Comment (py-rh) [reply docs/index.rst:14]: Hmm I don't super love it either way somehow lol

Comment (mkandler) [reply docs/index.rst:14]: It's pretty standard to go H1 -> content -> H2 -> more content

https://modal.com/docs/guide
https://python.langchain.com/docs/introduction/
https://docs.baseten.co/welcome
https://reactnative.dev/docs/getting-started
https://nextjs.org/docs (minimal but still)

File: examples/flux/readme.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Deploy Flux1 Schnell on AWS EC2
+
+See a more [rich explanation](https://www.run.house/examples/host-and-run-flux1-image-genai-aws)
```
Comment (mkandler) [inline examples/flux/readme.md:3]: If this isn't going in Examples, then this needs to change ☝️

File: docs/index.rst (hunk 1 above)
Comment (mkandler) [reply docs/index.rst]: Went down a rabbit hole questioning whether or not I've been thinking of this wrong, but feeling better with Google recommending not to have headers without text - https://developers.google.com/style/headings#heading-and-title-format

File: examples/flux/readme.md (hunk 4 above)
Comment (py-rh) [reply examples/flux/readme.md]: Good call! Changed to guides

**Pull request merged**